.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
Hmm we may want to tolerate things like `10.0.0` and `0.0.10000` and `0.0.1-beta`.

Reply:
`v10.0.0` and `v0.0.1000` should already work (these are globs, not regex).

It would be trivial to add `v0.0.1-beta` (`v*.*.*-beta`) in the future, if we decide to use that as some version that we want to create a release for. I'm somewhat hesitant to add it now, because it would open the door to bikeshedding whether or not we want to add `-alpha` or `-prerelease` or `-rc*` or things like that in the future. But feel free to tell me "just add it" and I'll do so.

Reply:
This also allows v0.0.1-beta anyway if it's a glob.